app/controllers/SignOutController.scala

```diff
@@ -54,21 +54,24 @@ class SignOutController @Inject() (
   }
 
   def doYouWantToGiveFeedback: Action[AnyContent] = Action { implicit request =>
-    Ok(views.doYouWantToGiveFeedbackPage(GiveFeedbackForm.form))
+    val taxRegimeString: String = request.session.get(SignOutController.feedbackRegimeKey).getOrElse("NA")
+    val maybeFallbackTaxRegime: Option[TaxRegime] = TaxRegime.withNameInsensitiveOption(taxRegimeString)
```

Comment:
if we can't get a tax regime here then something has gone wrong - you could extract out what we had before perhaps into a separate method and use it here and in the submit:
```scala
val taxRegimeString = request.session.get(SignOutController.feedbackRegimeKey)
    .getOrElse(sys.error("Could not find tax regime in cookie session"))
```

Reply:
Ok will do